examples/dark-mode/src/components/ThemeManager.astro

```diff
@@ -0,0 +1,69 @@
+---
+type Props = {
+  "defaultTheme"?: "auto" | "dark" | "light";
```

Comment:
```suggestion
  defaultTheme?: "auto" | "dark" | "light" | undefined;
```

Reply:
since it's optional I don't think we need undefined right but yeah removing the quotes for sure

Reply:
actually this can be needed when using the strictest tsconfig preset